CNV-57758: For 4.19.1-July 14th: Document virtualization with IBM fusion access for SAN (#92979)
openshift/openshift-docs: modules/virt-about-fusion-access-san.adoc

//
// * virt/fusion_access_SAN/about-fusion-access-san.adoc

:_mod-docs-content-type: CONCEPT
[id="about-fusion-access-san_{context}"]
= About {IBMFusionSAN}

{IBMFusionSAN} is a solution that provides a scalable clustered file system for enterprise storage, primarily designed to offer access to consolidated, block-level data storage. It presents storage devices, such as disk arrays and tape libraries, to the operating system as if they were direct-attached storage.

This solution is particularly geared towards enterprise storage for {VirtProductName} and leverages existing Storage Area Network (SAN) infrastructure. A SAN is a dedicated network of storage devices that is typically not accessible through the local area network (LAN).

.Why use {IBMFusionSAN}?

Easy user experience:: {IBMFusionSAN} features a wizard-driven user interface (UI) for installing and configuring storage clusters, file systems, and storage classes, to simplify the setup process.

Leverage existing infrastructure:: Organizations can leverage their existing SAN investments, including Fibre Channel (FC) and iSCSI technologies, as they transition to or expand with {VirtProductName}.

> Scalability:: The storage cluster is designed to scale with {product-title} clusters and virtual machine (VM) workloads. It can support up to approximately 3000 VMs on 6 bare-metal hosts, with possibilities for further scaling by adding more GPFS file systems or using specific storage class parameters.

Author: I just removed GPFS. It's now "with possibilities for further scaling by adding more file systems..."

Consolidated and shared storage:: SANs enable multiple servers to access a large, shared data storage capacity. This architecture facilitates automatic data backup and continuous monitoring of the storage and backup processes.

High-speed data transfer:: By using a dedicated high-speed network for storage, {IBMFusionSAN} overcomes the data transfer bottlenecks that can occur over a traditional LAN, especially for large volumes of data.

> File-level access:: Although a SAN primarily operates at the block level, file systems built on top of SANs provide file-level access, known as shared-disk file systems.

Reviewer: ```suggestion
File-level access:: Although a SAN primarily operates at the block level, file systems built on top of SAN storage can provide file-level access through shared-disk file systems..
```

// Module included in the following assemblies:
//
// * virt/fusion_access_SAN/about-fusion-access-san.adoc

:_mod-docs-content-type: CONCEPT
[id="about-fusion-access-san_{context}"]
= About {IBMFusionFirst}

{IBMFusionFirst} is a solution that provides a scalable clustered file system for enterprise storage, primarily designed to offer access to consolidated, block-level data storage. It presents storage devices, such as disk arrays, to the operating system as if they were direct-attached storage.

This solution is particularly geared towards enterprise storage for Red Hat {VirtProductName} and leverages existing Storage Area Network (SAN) infrastructure. A SAN is a dedicated network of storage devices that is typically not accessible through the local area network (LAN).

> [discrete]

Reviewer 2: Discrete headings are not supported in DITA. Since this is a concept module, this will need to become a full section. Looking at the contents of the section, I don't think it is a problem if it makes it in the table of contents.
```suggestion
```
